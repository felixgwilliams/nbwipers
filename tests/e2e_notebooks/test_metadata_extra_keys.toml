[tool.nbwipers]
extra-keys = ["metadata.kernelspec", "metadata.language_info"]
