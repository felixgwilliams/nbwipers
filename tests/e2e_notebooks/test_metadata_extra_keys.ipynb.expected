{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "This notebook tests that cells with either `\"keep_output\": true` or `\"init_cell\": true` are not stripped."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "init_cell": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "2"
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "1+1 # This cell has `\"init_cell:\" true`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "keep_output": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "4"
      ]
     },
     "execution_count": null,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "2+2 # This cell has `\"keep_output:\" true`"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": [
    "3+3"
   ]
  }
 ],
 "metadata": {
  "celltoolbar": "Edit Metadata"
 },
 "nbformat": 4,
 "nbformat_minor": 0
}
